01week/datatypes.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
+
+// Write a JavaScript program to display the current day and time.
+const todaysDate = new Date();
+ console.log(todaysDate);
+
+// Write a JavaScript program to convert a number to a string.
+const num = 15
+const a = num.toString();
+  console.log(a);
+
+// Write a JavaScript program to convert a string to the number.
+const text = '30px';
+const integer = parseInt(text, 10);
+  console.log(integer);
+
+// Write a JavaScript program that takes in different datatypes and prints out whether they are a:
+// Boolean
+console.log(typeof true);
+// Null
+console.log(typeof null)
+// Undefined
+console.log(typeof declaredButUndefinedVariable);
+
+// Number
+console.log(typeof 42);
+
+// NaN
+const isNaN = () => value === NaN;
+  console.log(NaN);
+
+// String
+console.log(typeof 'hello');
+
+// Write a JavaScript program that adds 2 numbers together.
+const SumTwoNumbers=(num1, num2)=>{
+  return num1 + num2
+};
+SumTwoNumbers(2, 2);
+
+// Write a JavaScript program that runs only when 2 things are true.
+const aﾠ = 1;
+const a = 2;
+
+if(aﾠ == 1 && a == 2) {
```

Comment:
this is an if/else statement that evaluates if 1 is equal to 1 and if 2 is equal to 2. I need a function that evaluates two arguments and if they are both truthy, runs a block of code.